.github/workflows/linters.yml

```diff
@@ -0,0 +1,48 @@
+name: Linters
```

Comment:
Please be sure that all of your linters are working properly 🧐 Lighthouse is passing all the checks but Webhint and Stylelint are failing. Kindly follow all the steps on this [link](https://github.com/microverseinc/linters-config/tree/master/html-css) to avoid issues. 

![Captura](https://user-images.githubusercontent.com/15807343/148119951-3ebc66b7-2e91-43ce-8916-9239dd010608.PNG)